guides/source/command_line.md

```diff
@@ -510,9 +510,9 @@ About your application's environment
 Rails version             7.2.0
 Ruby version              2.7.0 (x86_64-linux)
 RubyGems version          2.7.3
-Rack version              2.0.4
+Rack version              3.0.8
```

Comment:
Its worth for upgrading the rack version as the rails 7.1 default support the rack 3.